pkg/server/metrics/metrics.go

```diff
@@ -0,0 +1,88 @@
+package metrics
+
+import (
+	"net"
+	"net/http"
+	"strings"
+
+	grpcprom "github.com/grpc-ecosystem/go-grpc-middleware/providers/prometheus"
+	"github.com/prometheus/client_golang/prometheus"
+	"google.golang.org/grpc"
+	"k8s.io/component-base/metrics/legacyregistry"
+	"k8s.io/klog/v2"
+)
+
+var (
+	// metricsList is a list of all raw metrics that should be registered always, regardless of any feature gate's value.
+	metricsList       []prometheus.Collector
+	grpcServerMetrics *grpcprom.ServerMetrics
+)
+
+// Register registers a list of metrics.
+func Register() {
+	grpcServerMetrics = grpcprom.NewServerMetrics(
+		grpcprom.WithServerHandlingTimeHistogram(
+			grpcprom.WithHistogramBuckets([]float64{0.001, 0.01, 0.1, 0.3, 0.6, 1, 3, 6, 9, 20, 30, 60, 90, 120}),
+		),
+	)
+
+	metricsList = []prometheus.Collector{
+		grpcServerMetrics,
+	}
+
+	for _, metric := range metricsList {
+		legacyregistry.RawMustRegister(metric)
+	}
+}
+
+func ExportMetrics(metricsAddress string) {
```

Comment:
This method is setting up a metrics server, what about naming it as such? e.g. `SetupMetricsServer`